packages/react-frontend/src/routes/FooterFiles/Footer.jsx

```diff
@@ -0,0 +1,56 @@
+import { useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import "./Footer.css";
+
+
+export default function Footer(){
+    
+    useEffect(() => {
+        const handleLinkClick = () => {
+          window.scrollTo(0, 0); // Scroll to the top of the page whenever a link is clicked
+        };
+    
+        const links = document.querySelectorAll('a');
+        links.forEach(link => {
+          link.addEventListener('click', handleLinkClick);
+        });
+    
+        // Clean up the event listeners when the component unmounts
+        return () => {
+          links.forEach(link => {
+            link.removeEventListener('click', handleLinkClick);
+          });
+        };
+      }, []); 
+  return (
+    <footer>
+      <div className="footer-container">
+
+        <div className="footer-column">
+          <h3>Navigate</h3>
+          <Link to="/">Home</Link>
+          <Link to="/Albums">Albums</Link>
+        </div>
+        <div className="footer-column">
+          <h3>About Us</h3>
+          <Link to="/about">Our Story</Link>
+        </div>
+        <div className="footer-column">
+          <h3>FAQs</h3>
+          <Link to="/faqs">Click here</Link>
```

Comment:
I like how you separated out the footer into titles with the links beneath, but for the FAQ's since its one link, maybe it would be better to have it link directly to it instead of having the "Click Here" beneath it since "Click Here" is not a very telling/ meaningful title for the link.

Reply:
Thank you! Yes, I totally agree. I was thinking about maybe adding two different FAQ pages (such as faqs about the website and faqs about something else)!